dlls/vehicle.cpp

```diff
@@ -0,0 +1,996 @@
+#include "extdll.h"
```

Comment:
Maybe credit ReGameDLL_CS developers here?

Reply:
I'm not sure how it may look.